grails-app/conf/Config.groovy

```diff
@@ -262,6 +262,8 @@ environments {
 	}
 }
 
+streamr.ethereum.datacoinAddress = System.getProperty("streamr.ethereum.datacoinAddress") ? System.getProperty("streamr.ethereum.datacoinAddress") : "0x0cf0ee63788a0849fe5297f3407f701e122cc023"
```

Comment:
Hmm, I wonder why this token balance stuff is in this PR?

Reply:
I removed CORE-1187 code